packages/shared/src/utils/cfd/cfd.ts

```diff
@@ -250,76 +250,113 @@ export const isLandingCompanyEnabled = ({ landing_companies, platform, type }: T
-    const onfido_status = account_status?.authentication?.identity?.services?.onfido?.status;
-    const manual_status = account_status?.authentication?.identity?.services?.manual?.status;
+    const services = account_status?.authentication?.identity?.services ?? {};
+    const {
+        idv: { status: idv_status } = {},
+        onfido: { status: onfido_status } = {},
+        manual: { status: manual_status } = {},
+    } = services;
 
-    const acknowledged_status = ['pending', 'verified'];
-    const failed_cases = ['rejected', 'expired', 'suspected'];
+    const acknowledged_status: string[] = ['pending', 'verified'];
+    const failed_cases: string[] = ['rejected', 'expired', 'suspected'];
 
-    const poa_not_submitted = poa_status === 'none';
-    const poa_acknowledged = acknowledged_status.includes(poa_status);
-    const need_poa_submission = !poa_acknowledged;
-    const need_poa_resubmission = failed_cases.includes(poa_status);
-    const poa_verified = poa_status === 'verified';
-    const poa_pending = poa_status === 'pending';
+    const poa_not_submitted: boolean = poa_status === 'none';
+    const need_poa_submission = !acknowledged_status.includes(poa_status);
+    const need_poa_resubmission: boolean = failed_cases.includes(poa_status);
+    const poa_verified: boolean = poa_status === 'verified';
+    const poa_pending: boolean = poa_status === 'pending';
 
-    const poi_not_submitted = poi_status === 'none';
-    const poi_or_poa_not_submitted = poa_not_submitted || poi_not_submitted;
-    const poi_and_poa_not_submitted = poa_not_submitted && poi_not_submitted;
+    const poi_not_submitted: boolean = poi_status === 'none';
+    const poi_or_poa_not_submitted: boolean = poa_not_submitted || poi_not_submitted;
+    const poi_and_poa_not_submitted: boolean = poa_not_submitted && poi_not_submitted;
 
     //vanuatu-maltainvest
 
-    const poi_verified_for_vanuatu_maltainvest = [onfido_status, manual_status].includes('verified');
-    const poi_acknowledged_for_vanuatu_maltainvest =
-        (onfido_status && acknowledged_status.includes(onfido_status)) ||
-        (manual_status && acknowledged_status.includes(manual_status));
+    const requiredStatuses: (string | undefined)[] = [onfido_status, manual_status];
+    const submittedStatuses: string[] = requiredStatuses.filter((status: string | undefined) => status) as string[];
+    const allStatuses: (string | undefined)[] = submittedStatuses.concat([
+        idv_status,
+        onfido_status,
+        manual_status,
+    ] as string[]);
+
+    const poi_verified_for_maltainvest: boolean = allStatuses.includes('verified');
+    const poi_acknowledged_for_maltainvest: boolean = allStatuses.some((status: string | undefined) =>
+        acknowledged_status.includes(status ?? '')
+    );
+    const poi_pending_for_maltainvest: boolean =
+        allStatuses.every((status: string | undefined) => status === 'pending') && !poi_verified_for_maltainvest;
```

Comment:
I think here the check should be to verify if any one of the status is pending. So I should be `allStatuses.some`

Reply:
Got it. Thanks for noticing @likhith-deriv 
Will update